pkg/sources/gitlab/gitlab.go

```diff
@@ -10,6 +10,7 @@ import (
 	"sync"
 
 	"github.com/go-errors/errors"
+	git2 "github.com/go-git/go-git/v5"
```

Comment:
**Nit:** `gogit` or `gg` would be a better name imo

Reply:
haha hey man Go's auto-importer would disagree. But yea i like it. gg it is .